backend/docs/plantuml/sequenceInitialSubmission.puml

```diff
@@ -21,7 +21,7 @@
 
         processing -> backend: Query initial/edited sequence data
         backend -> DB: Query initial/edited sequence data
-        backend -> DB: Set status from 'RECEIVED'/'HAS_ERRORS' to 'IN_PROCESSING'
+        backend -> DB: Set status from 'RECEIVED' to 'IN_PROCESSING'
```

Comment:
I wonder whether the docs here were wrong in that one could also set status from "WAITING_FOR_APPROVAL" to "IN_PROCESSING", because that's what happened when one edited a sequence that passed warnings.

Reply:
Are we sure that status cannot go from 'PROCESSED' to 'IN_PROCESSING' here as well? Since we now call "HAS_ERRORS" "PROCESSED" this would be the logical change? So either this is a mistake or a bug fix of the original

Reply:
Never mind, it's just the old docs were wrong I think, this is easier to read and correct I think https://github.com/loculus-project/loculus/blob/5725b0b356529c788c6b3202cd9265ce8e9e79e8/backend/docs/plantuml/statusChange.svg

Reply:
Yeah I think if an edit is done, the PreprocessedData is deleted and then it is 'RECEIVED', and then the pipeline picks it up again and eventually moves it to IN_PROCESSING again.

resolving this then!